CHANGELOG.rst

```diff
@@ -1,6 +1,59 @@
 Changelog
 =========
 
+0.8.0 (2018-11-08)
+-----------------
+
+MLflow 0.8.0 introduces several major features:
+
+- Dramatically improved UI for comparing experiment run results:
+
+  - Metrics and parameters are by default grouped into a single column, to avoid an explosion of mostly-empty columns. Individual metrics and parameters can be moved into their own column to help compare across rows.
+  - Runs which are "nested" inside other runs (e.g., as part of a hyperparameter search or multistep workflow) now show up grouped by their parent run, and can be expanded or collapsed altogether.
```

Comment:
It might be nice to remind people how such nested runs are created: basically when you call mlflow.run inside an existing run.